### DISCUSSION TITLE
Error when adding the new encryption key.

### DISCUSSION THREAD

**Discussion by ewrible** [teslamate-org/teslamate#2758] [General]
So I got the message I needed to add an encryption key to my docker-compose.yml file, which I did.  When I bring down docker, I get the following:

eribble@NAS:/volume1/@docker$ sudo docker-compose down
ERROR: yaml.scanner.ScannerError: while scanning for the next token
found character '\t' that cannot start any token
  in "./docker-compose.yml", line 9, column 1

Thoughts?

**Comment by LelandSindt** [comment]
I _think_ that it means you have used a `tab` (\t) instead of spaces for indentation. 

https://yaml.org/faq.html

**Reply by ewrible** [reply]
Ok, time for me to go sit in a corner.  #ID10Terror
